src/service/routes_collections.py

```diff
@@ -192,6 +205,13 @@ async def save_collection(
     return sc
 
 
+@ROUTER_COLLECTIONS_ADMIN.get("/collectionids/all/", response_model = CollectionIDs)
```

Comment:
I'm not totally sold on the endpoint uris. If you have a better idea let me know